### PR TITLE
AW-131: Shapes with transparent borders do not render correctly

### PR DESCRIPTION
**Pull request by k1w1**
If the stroke/border is set to `transparent` color when do not render it at all.

### CODE REVIEW

File: dist/fabric.js [hunk 1]
```diff
@@ -17055,7 +17055,7 @@ fabric.util.object.extend(fabric.StaticCanvas.prototype, /** @lends fabric.Stati
      * @return {Object} .y height dimension
      */
     _getNonTransformedDimensions: function() {
-      var strokeWidth = this.stroke ? this.strokeWidth : 0,
+      (this.stroke && this.stroke !== 'transparent') ? this.strokeWidth : 0,
```
Comment (gregplaysguitar) [inline dist/fabric.js]: Seem to have lost the `strokeWidth` variable here?

File: src/mixins/object_geometry.mixin.js [hunk 2]
```diff
@@ -639,7 +639,7 @@
      * @return {Object} .y height dimension
      */
     _getNonTransformedDimensions: function() {
-      var strokeWidth = this.stroke ? this.strokeWidth : 0,
+      (this.stroke && this.stroke !== 'transparent') ? this.strokeWidth : 0,
           w = this.width + strokeWidth,
           h = this.height + strokeWidth;
       return { x: w, y: h };
```
Comment (gregplaysguitar) [inline src/mixins/object_geometry.mixin.js]: Missing `strokeWidth` here too

**Review by gregplaysguitar — CHANGES_REQUESTED** [review]
Makes sense, just seem to have lost a variable - see comments

File: src/mixins/object_geometry.mixin.js (hunk 2 above)
Comment (k1w1) [reply src/mixins/object_geometry.mixin.js]: Looks like I accidentally deleted it when fixing the formatting. Fixed now.

**Review by gregplaysguitar — APPROVED** [review]
👍 

It looks like Jeremy's changes from https://github.com/aha-app/fabric.js/pull/11 are included here too - the dist files weren't rebuilt in that PR

**Pull request merged**